tests/security/ebpf/check_cap_bpf.pm

```diff
@@ -12,17 +12,38 @@ use testapi;
 use utils;
 
 sub run {
-    my $f_tcpdump = '/usr/sbin/tcpdump';
     my $capability = 'cap_bpf';
+    my $f_bpf_test = '/tmp/bpf_test';
 
     select_console 'root-console';
+    # Set 'unprivileged_bpf_disabled' to 1
+    validate_script_output('sysctl kernel.unprivileged_bpf_disabled=1', sub { m/kernel.unprivileged_bpf_disabled = 1/ });
+    validate_script_output("cat /proc/sys/kernel/unprivileged_bpf_disabled", sub { m/1/ });
 
-    # Install packages
-    zypper_call('in libcap-progs tcpdump');
+    # Download the C test code and compile
+    assert_script_run('zypper -n in gcc libcap-progs', timeout => 300);
+    assert_script_run('wget ' . autoinst_url . '/data/ebpf/bpf_test.c');
+    assert_script_run("gcc -o $f_bpf_test bpf_test.c");
+    # BPF system call should succeed with root permission
+    validate_script_output("$f_bpf_test", sub { m/BPF: Success/ });
 
-    assert_script_run("getcap $f_tcpdump");
-    assert_script_run("setcap $capability+eip $f_tcpdump");
-    validate_script_output("getcap $f_tcpdump", sub { m/.*$capability.*/ });
+    # Delete the script file `validate_script_output` just created,
+    # to avoid the permission denied issue in common user.
```

Comment:
Clean up should be better, and please add some description why the files are created

Reply:
Updated, thanks.